src/schemaValidation/validateSchema.ts

```diff
@@ -0,0 +1,169 @@
+import { CliError } from "../params";
+import { ReleaseFileType } from "../types";
+import Ajv, { ErrorObject } from "ajv";
+import ajvErrors from "ajv-errors";
+// Schemas
+import manifestSchema from "./schemas/manifest.schema.json";
+import composeSchema from "./schemas/compose.schema.json";
+import setupWizardSchema from "./schemas/setup-wizard.schema.json";
+import { SetupWizard } from "../releaseFiles/setupWizard/types";
+import { Manifest } from "../releaseFiles/manifest/types";
+import { Compose } from "../releaseFiles/compose/types";
+
+const ajv = new Ajv({
+  allErrors: true,
+  coerceTypes: true,
+  strictSchema: false
+});
+
+ajvErrors(ajv);
+
+console.log("Validating manifest schema");
+const manifestValidate = ajv.compile(manifestSchema);
```

Comment:
I think ajv.complie caches the result. Doing it here will require compiling schemas if you just want to --help. I think it's okay to do it on demand

Reply:
makes sense, now compiling on demmand